src/main/resources/geoserver/layer_group_creation.xml

```diff
@@ -0,0 +1,27 @@
+<layerGroup>
+    <name>LAYER_GROUP_NAME</name>
+    <mode>SINGLE</mode>
+    <title>Beachfront Detections</title>
+    <workspace>
+        <name>piazza</name>
```

Comment:
Workspace name is in the configuration but this looks like a literal

Reply:
Good catch!
Should be fixed.